doc/lua_api.txt

```diff
@@ -2424,6 +2424,11 @@ Helper functions
 * `table.insert_all(table, other_table)`:
     * Appends all values in `other_table` to `table` - uses `#table + 1` to
       find new indices.
+* `table.shuffle(table, [from], [to], [random_func])`:
+    * Shuffles elements `from` to `to` in `table` in place
+    * `from` defaults to `1`
+    * `to` defaults to `#table`
+    * `random_func` defaults to `math.random`
```

Comment:
I think the documentation should describe what the function parameter should do. it doesn't look like it's required to provide the full functionality of math.random

Reply:
You marked this resolved but you didn't make any related change or respond with why it's unnecessary.